### DISCUSSION TITLE
TensorRT python binding

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#97] [General]
Hello,
Does the Nvidia Yocto supports the tensorrt python binding?
I am looking for the tensorrt.py file and cannot find it.
Do I need to select a specific package for the binding to occur?

Thanks

**Comment by tirankas** [comment]
Additionally, I dont see the tensorrt.so file as part of the build artifacts, although tensorrt is part of the image packages.

**Comment by madisongh** [comment]
See https://github.com/OE4T/meta-tegra/discussions/701
